.github/workflows/performance.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - run: echo "$GITHUB_CONTEXT"
       - run: |
           # Create and Push Branch
-          git clone https://jlb6740:${{env.TOKEN}}@github.com/bytecodealliance/wasmtime-sightglass-benchmarking.git
+          git clone https://wasmtime-publish:${{secrets.PERSONAL_ACCESS_TOKEN}}@github.com/bytecodealliance/wasmtime-sightglass-benchmarking.git
```

Comment:
If this is referencing `secrets` (which I agree is what should be done here) I think the `TOKEN` above is no longer referenced and can be deleted?